File: 백준/Silver/2563. 색종이/색종이.cc
#include <iostream>
#include <vector>
using namespace std;

int main(void) {
	ios_base::sync_with_stdio(false);
	cin.tie(nullptr);

	int pcount = 0;
	if (!(cin >> pcount)) return 0;

	vector<vector<int>> paper(100, vector<int>(100, 0));

	for (int i = 0; i < pcount; i++) {
		int x = 0, y = 0;
		if (!(cin >> x >> y)) return 0;

		for (int j = x; j < x + 10; j++) {
			for (int k = y; k < y + 10; k++) {
				paper[j][k] = 1;
			}
		}
	}

	int total = 0;
	for (int i = 0; i < 100; i++) {
		for (int j = 0; j < 100; j++) {
			total += paper[i][j];
		}
	}

	cout << total << "\n";

	return 0;
}

File: 백준/Silver/2563. 색종이/색종이.py
import sys
input = sys.stdin.readline

def solve():
    pcount = int(input().rstrip())
    if not pcount: return
    
    paper = [[0] * 100 for _ in range(100)]
    for _ in range(pcount):
        line = input().split()
        if not line: return
        x, y = map(int, line)
        
        for i in range(x, x + 10):
            for j in range(y, y + 10):
                paper[i][j] = 1
    
    total = 0
    for i in paper:
        total += sum(i)
        
    print(total)       
        
if __name__ == "__main__":
    solve()